src/common/string-builder.ts

```diff
@@ -54,7 +54,16 @@ class StringBuilder {
     }
 
     makeParams(params?: object) {
-        if(this.config.params && params) this.printQueue.push(JSON.stringify(params));
+        if(this.config.params && params) {
+            let str: string = '';
+            if (isURLSearchParams(params)) {
+                const obj = convertURLSearchParamsToObject(params as URLSearchParams);
+                str = JSON.stringify(obj);
```

Comment:
If the isURLSearchParams function acts as a type guard, it seems there would be no need to use the as keyword in the following statement. Could you improve this part?